src/openrct2/scripting/ScriptEngine.h

```diff
@@ -46,7 +46,7 @@ namespace OpenRCT2
 
 namespace OpenRCT2::Scripting
 {
-    static constexpr int32_t OPENRCT2_PLUGIN_API_VERSION = 63;
+    static constexpr int32_t OPENRCT2_PLUGIN_API_VERSION = 64;
```

Comment:
Is this necessary if API is only added to?

Reply:
It allows plugins to easily check if one or more changes are available / present.

Reply:
So, yes?

Reply:
Yes